### PR TITLE
Feat/learningdataloader

### PR DESCRIPTION
**Pull request by birproject**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you
did and what alternatives you considered, etc...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<h1 align="center">Hexagonal Architecture Typescript Service Skeleton</h1>
+<h1 align="center">Super GraphQL Backend with the best Practices in the World</h1>
```
Comment (Juanfe98) [inline README.md:1]: Please Do not add these kind of things to the project

File: src/apolloContexters.ts [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
+// @ts-ignore
+let tokenCounter = 0
+
+
+// Create DataLoaders batching and caching graphql requests
+function createAdLoader() {
+  const adLoader = new DataLoader(async (ids: readonly string[]) => {
+    console.log('Analyzing batch function 🔥🔥🔥🔥🔥🔥', ids)
+    if (!adsRepository) {
+      console.log('🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯 Could instantiate the adsRepository properly')
```
Comment (Juanfe98) [inline src/apolloContexters.ts:28]: Create a real Logger for this

File: src/apolloContexters.ts [hunk 3]
```diff
@@ -1,14 +1,52 @@
...
+    }
+    const ads = await adsRepository.getAdsById(ids)
+    const adsMap = new Map(ads.map(ad => [ad.id.toString(), ad]))
+    return ids.map(id => adsMap.get(id) || new Error(`Ad not found for ID: ${id}`))
+  })
+  // console.log('🫡 Ad loader loaded successfully times = ', loaderCounter++)
```
Comment (Juanfe98) [inline src/apolloContexters.ts:34]: Remove unnecesary lines

**Review by Juanfe98 — DISMISSED** [review]

File: src/apolloContexters.ts (hunk 2 above)
Comment (birproject) [reply src/apolloContexters.ts:28]: shut up stupid

Comment (birproject) [reply src/apolloContexters.ts:28]: I do what I want cause I am super owner, your comment is not relevant keep working employee

**Review by cristiantorresf — APPROVED** [review]
I love it

File: src/apolloContexters.ts (hunk 2 above)
Comment (Juanfe98) [reply src/apolloContexters.ts:28]: MTF

**Pull request merged**